platform/application.go

```diff
@@ -21,6 +21,7 @@ import (
 )
 
 // ApplicationConfig represents a minimal API configuration that can be override / augmented by the application
+// Deprecated: This function should no longer be used. Use the API servers instead.
```

Comment:
👍

Reply:
is it raising warning at compile time ?

Reply:
ouuuh little squiggly lines ! 
A bit sad that I filled the golang survey already ^^